GMusicProxy

```diff
@@ -318,20 +318,44 @@ class GetHandler(BaseHTTPServer.BaseHTTPRequestHandler):
             if tagsBin and not do_shoutcast:
                 self.wfile.write(tagsBin)
 
+            # prefill cache
+            cachePrefillSize = 60 * 320/8 * 1024 # 60 seconds at 320 kbit/s
+            writtenBytes = 0
             mp3 = opener.open(url)
             block = mp3.read(downloadBlockSize)
             while len(block) > 0:
                 self.wfile.write(block)
+                writtenBytes += len(block)
                 if do_shoutcast:
                     self.wfile.write(self._icy_metadata(
                         tags.album, tags.artist, tags.title))
                 block = mp3.read(downloadBlockSize)
+                if writtenBytes > cachePrefillSize:
+                    break
 
             if not config['disable_playcount_increment']:
                 logger.info('Increment playcount')
                 self._robust_retry(
                     lambda: api.increment_song_playcount(song_id=id))
 
+            # consume stream
+            tempFile = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
why are you not using the auto-delete feature (on by default) of the python temporary file support?